internal/gapicgen/cmd/genbot/github.go

```diff
@@ -240,10 +240,12 @@ func (gc *GithubClient) CreateGocloudPR(ctx context.Context, gocloudDir string,
 	log.Println("creating google-cloud-go PR")
 
 	var body string
+	draft := github.Bool(true)
```

Comment:
Consider making this a regular `bool` then converting it with `github.Bool` when calling `Create`.